grazie/src/main/kotlin/org/rust/grazie/RsGrammarCheckingStrategy.kt

```diff
@@ -53,7 +54,7 @@ class RsGrammarCheckingStrategy : GrammarCheckingStrategy {
     }
 
     override fun getContextRootTextDomain(root: PsiElement): TextDomain {
-        return when (root.elementType) {
+        return when (root.elementTypeOrNull) {
```

Comment:
What's about other usages of `elementType`? For example, in `getRootsChain `

Reply:
I didn't see an exception there, but I can easily replace it it as well